organizations/v0/views.py

```diff
@@ -25,6 +25,7 @@ class OrganizationsViewSet(mixins.UpdateModelMixin, viewsets.ReadOnlyModelViewSe
     """
     queryset = Organization.objects.all()
     serializer_class = OrganizationSerializer
+    lookup_value_regex = '[^/+]+'   # the default regex doesn't work if there is a period inside short name
```

Comment:
This comment doesn't really match what's going on in the overridden regex. The way I read the regex, it means "one or more of any character that's not a plus sign" - is that correct? Why exclude the plus sign? Why not instead a regex like `[\w\-\.]+` - which more clearly means "one of more of alphanumeric, dash, and dot characters"?

Reply:
Thanks for the explanation, @waheedahmed . Could you please put a one-line comment to this effect above the regex? Something like:
`# Include everything not a '+' - identical to discovery's COURSE_ID_REGEX.`

Reply:
Done!